### ISSUE TITLE
API always returning success message even when connection fails

### ISSUE THREAD

**Issue by baty0man** [OpenWonderLabs/SwitchBotAPI#189]

**Comment by baty0man** [comment]
Still an issue

**Comment by baty0man** [comment]
Still an issue

**Comment by baty0man** [comment]
Still an issue

**Comment by baty0man** [comment]
Still an issue

**Comment by edsaavedra84** [comment]
I agree, this is really annoying making the API somewhat unreliable, I have the exact same problem as you, I even have the switchbot hub very close to a press and curtain bot, and sometimes I get the described connection error. 

![Screenshot_20230905-231825.png](https://github.com/OpenWonderLabs/SwitchBotAPI/assets/59943172/c16f080e-21ed-4558-bf68-c779b000fedd)

I suppose I can check the status of the curtain after a rest call to do a retry, but for the Press bot its a deal breaker since I use it in button presser mode

**Comment by davidsak** [comment]
I’m having this issue. It’s happened in the past and then stopped and just recently started again. It’s happening across several hubs all when trying to control an air conditioner device.
